docs/best/react.md

```diff
@@ -194,7 +194,7 @@ message.likes.push("Jennifer")
 ```
 
 This will react with the above sample data, array indexers count as property access. But **only** if the provided `index < length`.
-MobX will not track not-yet-existing indices or object properties (except when using maps).
```

Comment:
The below code will print twice. So I think the not-yet-existing object properties can be tracked.

```
import { observable, autorun } from "../mobx"

const message = observable({
    likes: {},
}) as any

autorun(() => {
    console.log(message.likes[0])
})
message.likes[0] = "Jennifer"
```

Reply:
yeah, this is old and only applies to non proxy environments, so the change is correct.